src/frontend/src/test-e2e/selenium.test.ts

```diff
@@ -106,9 +106,15 @@ test("Log into client application, after registration", async () => {
     await demoAppView.waitForDisplay();
     const principal = await demoAppView.getPrincipal();
     expect(principal).not.toBe("2vxsx-fae");
+
+    /*
+    on dfx > 0.8.0 the woami container does not respond
+    TODO: this should be fixed asap
```

Comment:
Can we have a ticket and link to said ticket here?

Reply:
done https://dfinity.atlassian.net/browse/II-74